getting-started/installation-source-windows.md

```diff
@@ -26,6 +26,9 @@ you should be able to open the folder in Visual Studio, which will take
 care of the rest.
 
 If you are using an earlier version of Visual Studio:
+
+>:TIP: This install step has to be made as admin.
```

Comment:
Hmm.. Maybe make this a warning? Otherwise, LGTM.

Reply:
I agree